public/javascripts/export1Table.js

```diff
@@ -0,0 +1,119 @@
+var pickTranslation = function(songID, translationID, checked) {
+    if (checked) {
+        console.log(translationID)
+        m.request({
+                method: 'POST',
+                url: '/user/playlist/' + playlistName + '/export1',
+                data: {
+                    playlistID: playlistID,
+                    songID: songID,
+                    translationID: translationID
+                }
+            })
+            .then(function(res) {
+                console.log(res.msg)
+            })
+    } else {
+        m.request({
+                method: 'DELETE',
+                url: '/user/playlist/' + playlistName + '/export1',
+                data: {
+                    playlistID: playlistID,
+                    songID: songID,
+                    translationID: translationID
+                }
+            })
+            .then(function(res) {
+                console.log(res.msg)
+            })
+    }
+
+}
+
+
+var langLabelArr = function(translationss) {
+    var lang = []
+    translationss.forEach((tr) => {
+        tr.forEach((t, i) => {
+            if (!_.includes(lang, t.lang)) {
+                lang.push(t.lang)
+            }
+        })
+    })
+    return lang
+}
+
+var langOptions = langLabelArr(translationss)
+
+
+var selectAll = function(elem, checkboxClass) {
+    $(elem).click(function() {
+        $('.' + checkboxClass).trigger('click')
+    });
+}
+
+$(window).load(function() {
+    langsPicked.forEach((lp) =>
+        lp.forEach((l) => {
+            $('#' + l).prop('checked', true)
+        }))
+})
+
+
+var export1Table = {
+    view: function() {
+        return [
+            m('table.table', [
+                m('thead', [
+                    m('th', 'Title'),
+                    langOptions.map((l) => {
+                        return m('th', l)
+                    })
+                ]),
+                m('tbody', [
+                    m('tr', [
+                        m('td'),
+                        langOptions.map((l) => {
+                            return m('td', [
+                                m('button.btn.btn-default.btn-sm', {
+                                    config: function(elem, isInit, context) {
+                                        if (!isInit) {
+                                            selectAll(elem, l);
+                                        }
+                                    }
+                                }, 'Select/Deselect All')
```

Comment:
I'd recommend actually just making this another checkbox. I think people intuitively know that checking it off will select/de-select all in the column and we could have the explanatory text appear as a tooltip (title="Select/Deselect All") instead.